app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -4,7 +4,7 @@ module Api
   module V1
     class LobbiesController < AuthenticatedController
       before_action :require_authentication, except: [:from_code, :join]
```

Comment:
`answer` should not require authentication, because "guest" players can also take part in quizzes 😄